repose-aggregator/components/filters/ip-classification/src/main/scala/edazdarevic/commons/net/CIDRUtils.java

```diff
@@ -0,0 +1,142 @@
+/*
```

Comment:
you have a java class in the scala source folder, also why did we copy pasta this?